crates/egui/src/input_state.rs

```diff
@@ -393,6 +393,16 @@ impl InputState {
         self.consume_key(modifiers, logical_key)
     }
 
+    /// Verifies that all modifier keys (e.g. Ctrl, Alt) are not pressed.
+    pub fn modifiers_not_pressed(&self) -> bool {
+        self.modifiers.is_none()
+    }
+
+    /// Was only the given key pressed this frame, with no modifiers?
+    pub fn key_pressed_only(&self, desired_key: Key) -> bool {
+        self.num_presses(desired_key) > 0 && self.modifiers_not_pressed()
```

Comment:
if one day `key_pressed` might be modified -we could easily forget this - fix this using the function here, too 

```rust 
pub fn key_pressed_only(&self, desired_key: Key) -> bool {
        key_pressed(desired_key) && self.modifiers_not_pressed()
}
```